<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeStatistics</class>
 <widget class="QMainWindow" name="CodeStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeStatistics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFileCount">
        <property name="text">
         <string>总文件数:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fileCountEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelCodeCount">
        <property name="text">
         <string>总代码行数:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QLineEdit" name="codeCountEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelCodePercent">
        <property name="text">
         <string>[0%]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelByteCount">
        <property name="text">
         <string>总字节数:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLineEdit" name="byteEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelCommentCount">
        <property name="text">
         <string>总注释行数:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="2">
       <widget class="QLineEdit" name="commentEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="labelCommentPercent">
        <property name="text">
         <string>[0%]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="labelLineCount">
        <property name="text">
         <string>总行数:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="labelBlankCount">
        <property name="text">
         <string>空白行数:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="blankEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="labelBlankPercent">
        <property name="text">
         <string>[0%]</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenDir"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionClearModel"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>语言</string>
     </property>
     <addaction name="actionCN"/>
     <addaction name="actionEN"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>编码</string>
     </property>
     <addaction name="actionUTF8"/>
     <addaction name="actionGBK"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_T"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClearModel">
   <property name="text">
    <string>清空列表</string>
   </property>
  </action>
  <action name="actionCN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionEN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionGBK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GBK</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
